src/plugins/accounting.cpp

```diff
@@ -332,11 +332,22 @@ bool Association::under_max_resources (const Job &job)
     return under_max_resources;
 }
 
-bool Association::under_queue_max_resources (const Job &job,
-                                             const Queue &queue)
+bool Association::under_queue_max_resources (
+                                    const Job &job,
+                                    const std::string &queue,
+                                    const std::map<std::string, Queue> queues)
```

Comment:
Another C++ newbie question: should `queues` be passed by reference here to avoid a copy?